doc/lua_api.md

```diff
@@ -4111,9 +4111,9 @@ On some specific cases, server translation could be useful. For example, filter
 a list on labels and send results to client. A method is supplied to achieve
 that:
 
-`minetest.get_translated_string(lang_code, string)`: Translates `string` using
-translations for `lang_code` language. It gives the same result as if the string
-was translated by the client.
+`minetest.get_translated_string(lang_code, S(string))`: returns the translation in
+`lang_code` language of `S(string)`. It gives the same result as if the string was
+translated by the client.
```

Comment:
my suggestion to keep it exact while still resolving the confusion:
```suggestion
`minetest.get_translated_string(lang_code, S(string))`: resolve translations in the given string
just like the client would, using the translation files for `lang_code`.
For this to have any effect the string has to contain what was returned by `minetest.translate`.
```

Reply:
Idk, I find mine more explicative but I'm not a native speaker. @rubenwardy ?

(also, *resolves)

Reply:
sfan5's is more accurate, although I'd maybe change the last line to

> For this to have any effect, the string needs to contain translation markup (as made by `minetest.translate`, `get_translator`, or `S()`).